instrumentation/kafka/src/test/java/brave/kafka/ITTracingKafka.java

```diff
@@ -0,0 +1,161 @@
+package brave.kafka;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import brave.Tracing;
+import brave.sampler.Sampler;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.Properties;
+import java.util.concurrent.ConcurrentLinkedDeque;
+import java.util.concurrent.Future;
+import kafka.admin.AdminUtils;
+import kafka.server.KafkaConfig;
+import kafka.server.KafkaServer;
+import kafka.utils.TestUtils;
+import kafka.utils.ZKStringSerializer$;
+import kafka.utils.ZkUtils;
+import kafka.zk.EmbeddedZookeeper;
+import org.I0Itec.zkclient.ZkClient;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.utils.Time;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import zipkin.Span;
+
+public class ITTracingKafka {
+
+    private static final String LOCALHOST = "127.0.0.1";
```

Comment:
nit I tend to use just normal instance fields in junit tests as it is less distracting to read.